app/Http/Controllers/API/DiscourseController.php

```diff
@@ -19,20 +19,25 @@ public function discussionTopics(Request $request, DiscourseService $discourseSe
     {
         $topics = [];
 
+        if (!config('restarters.features.discourse_integration')) {
+            return response()->json([
+                'success' => 'success',
+                'topics' => $topics
+            ], 200);
```

Comment:
This seems odd as a stub? Maybe a 204 (No Content) or a 400 code could be more appropriate? (not a blocker, just asking for my own understanding, there's nothing about it in the commit message or description)

Reply:
Addressed in [db4efd3](https://github.com/iFixit/restarters/pull/11/commits/db4efd376e1cd6663a981e9aa1a8a2fa5b8e4a48)